docs/source/configuration.rst

```diff
@@ -83,6 +96,17 @@ Also modify ``Dockerfile`` to include the following lines::
 
     RUN pip install django_ses
 
+- Testing and debugging e-mail configuration:
+
+Sending test e-mail to one or more addresses::
+
+    ./manage.py sendtestemail user1@example1.tld user2@example2.tld ...
+
+Sending test e-mail to the administrators::
+
+    ./manage.py sendtestemail --admins
```

Comment:
Don't describe individual options to `sendtestemail`. Instead point to upstream documentation. https://docs.djangoproject.com/en/3.0/ref/django-admin/#sendtestemail